.vscode/launch.json

```diff
@@ -21,7 +21,7 @@
       "request": "launch",
       "console": "internalConsole",
       "preLaunchTask": "build",
-      "program": "${workspaceRoot}/tools/converters/excel/bin/Debug/net5.0/Pims.Tools.Converters.ExcelConverter.dll",
+      "program": "${workspaceRoot}/tools/converters/excel/bin/Debug/net6.0/Pims.Tools.Converters.ExcelConverter.dll",
```

Comment:
This whole section can be safely removed - we don't have the excel converter in the solution anymore

Reply:
Removed !